rest/indextest/index_init_api_test.go

```diff
@@ -123,18 +125,17 @@ func TestChangeIndexPartitions(t *testing.T) {
 						assert.Equal(c, db.CollectionIndexStatusReady, status)
 					}
 				}
-			}, 30*time.Second, 1*time.Second)
+			}, 1*time.Minute, 1*time.Second)
```

Comment:
Did you find this really takes a long time? I find server is slow and if you have multiple nodes it is really slow.

Is it worth polling more frequently than 1 sec to make the test faster, or do these take 20-30 secs?

Because this is so unexpected, I might add partitionIndexPollingTime, partitionIndexPollingExpiration for times since this is duplicated.

Reply:
It's usually reasonably quick (~7 sec locally), but I saw it fail once on Jenkins where it took slightly longer than 30sec 

I don't think it's worth polling more frequently because at most you save 1 second, but there's also the interaction between this test retry loop and the `WaitForIndexesOnline` backoff retry loop which can sleep for much longer.